spec/chromium-spec.ts

```diff
@@ -655,6 +656,8 @@ describe('chromium features', () => {
     });
 
     it('should lock the keyboard', async () => {
+      const screenCapture = new ScreenCapture();
+      await screenCapture.takeScreenshot('keyboardlock');
```

Comment:
just checking to be sure, should this be removed prior to merging?

Reply:
@samuelmaddock - yes.  I want to verify that the change is actually working.